packaging/debian/postinst

```diff
@@ -1,12 +1,16 @@
 #!/bin/sh
 
 #DEBHELPER#
-mkdir -p /var/log/percona/telemetry-agent
-chown daemon:percona-telemetry /var/log/percona/telemetry-agent
-chmod 775 /var/log/percona/telemetry-agent
-systemctl daemon-reload
-systemctl enable percona-telemetry-agent.service || true
-systemctl start percona-telemetry-agent.service
+# Create log directory
+install -d -m 0775 -o daemon -g percona-telemetry /var/log/percona/telemetry-agent
+install -D -m 0660  -o daemon -g percona-telemetry /dev/null /var/log/percona/telemetry-agent/telemetry-agent.log
+install -D -m 0660  -o daemon -g percona-telemetry /dev/null /var/log/percona/telemetry-agent/telemetry-agent-error.log
+# Move the old log-files into the new directory
+if [ -e /var/log/percona/telemetry-agent.log ] && [ -e /var/log/percona/telemetry-agent-error.log ]; then
```

Comment:
I would check each log file separately, I mean 2 `if` blocks

Reply:
I have changed the "&&" condition to "||", so it's evaluated even if one of the log file is present